inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/controller/InlongGroupController.java

```diff
@@ -252,4 +252,11 @@ public Response<Boolean> finishTagSwitch(@PathVariable String groupId) {
         return Response.success(groupService.finishTagSwitch(groupId));
     }
 
+    @RequestMapping(value = "/group/submitOfflineJob/{groupId}", method = RequestMethod.POST)
+    @ApiOperation(value = "Submitting inlong offline job process")
+    @ApiImplicitParam(name = "groupId", value = "Inlong group id", dataTypeClass = String.class)
+    public Response<WorkflowResult> submitOfflineJob(@PathVariable String groupId) {
+        String operator = LoginUserUtils.getLoginUser().getName();
+        return Response.success(groupProcessOperation.restartProcess(groupId, operator));
```

Comment:
restart rather than start ?

Reply:
this should be the `groupService.submitOfflineJob`